src/components/map-controls/map-3d.vue

```diff
@@ -43,8 +95,21 @@ const toggle3d = () => {
       mapStore.is3dActive ? 'active' : ''
```

Comment:
Is there more code related to 3D that can be deleted or refactored in v4? Not sure why we already had a 3D button, I cannot remember that we already migrated anything.